src/features/projects/api/projects.ts

```diff
@@ -0,0 +1,50 @@
+import axiosInstance from '@/lib/axios'
+
+export const getProjects = async () => {
+  const response = await axiosInstance.get('/projects')
+  return response.data
+}
+
+export const createProject = async (
+  name: string,
+  teamId: number,
+  schemaId: number
+) => {
+  const response = await axiosInstance.post('/projects', {
+    name,
+    team_id: teamId,
+    schema_id: schemaId,
+  })
+  return response.data
+}
+
+export const getDocumentsByProject = async (projectId: number) => {
+  const response = await axiosInstance.get(`/projects/${projectId}/documents`)
+  return response.data
+}
+
+export const createDocument = async (
+  projectId: number,
+  fileName: string,
+  fileContent: string
+) => {
+  const response = await axiosInstance.post(
+    `/projects/${projectId}/documents`,
+    { name: fileName, content: fileContent }
+  )
+  return response.data
+}
+
+export const deleteDocument = async (documentId: number) => {
+  await axiosInstance.delete(`/documents/${documentId}`)
+}
+
+export const getTeams = async () => {
+  const response = await axiosInstance.get('/teams/')
```

Comment:
```suggestion
  const response = await axiosInstance.get('/teams')
```
Eigentlich müsste es so aussehen. Das ist wieder Bad Practice aus dem Backend...

Bei allen anderen gleichen Request natürlich genauso. Hab keine Lust das alles zu markieren :)